server/compute.js

```diff
@@ -46,6 +47,7 @@ module.exports = function compute({ changes = {}, user, segments, events = [] },
   sandbox.ship = ship;
   sandbox.payload = {};
   sandbox.isInSegment = isInSegment.bind(null, segments);
+  sandbox.Hull = Hull;
```

Comment:
Good point @unity, didn't think about this while making the changes, but you're right, it's too dangerous. I am going to change it and expose only a logInfo function. @sbellity, thanks for the heads up, let's discuss later...